src/markdown-pages/why-galasa.md

```diff
@@ -0,0 +1,124 @@
+---
+path: "/about/different"
+title: "Why Galasa is different"
+---
+
+What makes Galasa different to other testing tools?
+
+If you've ever struggled to implement automated testing across a complex technology stack, you might recognize some of the issues that we identified during our design process. Expand the following headings to see how Galasa solves these challenges.
+
+## Running consistent, repeatable, reliable tests across the application stack: 
+
+
+<details>
+<summary><b>Ensuring consistent testing across technologies</b></summary>
+
+Automating tests effectively and consistently across multiple environments and multiple tools requires a framework that can support and integrate across the whole application stack. The same test must also be capable of running against multiple z/OS environments without code changes.
+
+Galasa is designed to work in hybrid cloud applications, operating with the entire application stack, so you can  provision in a cloud environment, but bind to a z/OS test environment.
+The Galasa framework works across multiple technologies in a consistent way, providing a platform to test different aspects of the application stack in the same language and in the same fashion. One test case can interact with 3270 emulators, Selenium, JMeter, batch jobs and other applications. 
+</details>
+
+<details>
+<summary><b>Verification and validation without mocking or stubbing</b></summary>
+
+Verifying and validating data at every boundary without stubbing or mocking is hard. 
+Galasa enables real data verification by interrogating a CICS (or other) application directly or by enabling the checking of other z/OS resources, for example, messages on queues or the updating of log streams.
+</details>
+
+<details>
+<summary><b>Reliability and availability of test data</b></summary>
+
+Test data is often in a state of flux, resulting in the breaking of existing tests and difficulty in snapshotting and data integrity.
+
+Galasa enables you to provision your own test data from scratch or find valid test data within an existing data lake. Test data is locked within the Galasa framework whilst in use, so that it cannot be corrupted by other test runs.
```

Comment:
Might want to get @hobbit1983 opinion on this, but this reads to me that galasa will provision test data for you. Where galasa provides the mechanisms to allow you to generate test data on the fly via an application (or similar) manager, galasa is not doing this work for you. Think we need to be careful here to set expectations correctly.
Same is true for data locking, it has to be provided as part of a applicaiton specific manager.